--- a/pyximport/pyximport.py	2023-01-05 23:13:06.749019000 +0100
+++ b/pyximport/pyximport.py	2023-05-03 10:36:50.798654635 +0200
@@ -541,7 +541,10 @@
     if setup_args is None:
         setup_args = {}
     if not build_dir:
-        build_dir = os.path.join(os.path.expanduser('~'), '.pyxbld')
+        if 'PYXBLD_DIR' in os.environ:
+            build_dir = os.environ['PYXBLD_DIR']
+        else:
+            build_dir = os.path.join(os.path.expanduser('~'), '.pyxbld')
 
     global pyxargs
     pyxargs = PyxArgs()  #$pycheck_no
