--- a/tests/test_htmlhelp.py	2023-01-18 16:00:43.901935575 +0100
+++ b/tests/test_htmlhelp.py	2023-01-18 16:00:08.950343305 +0100
@@ -21,7 +21,7 @@
 def test_build_htmlhelp(app, status, warning):
     app.build()
 
-    hhp = (app.outdir / 'pythondoc.hhp').text()
+    hhp = (app.outdir / 'pythondoc.hhp').read_text()
     assert 'Compiled file=pythondoc.chm' in hhp
     assert 'Contents file=pythondoc.hhc' in hhp
     assert 'Default Window=pythondoc' in hhp
@@ -84,7 +84,7 @@
         assert node[1].attrib == {'name': 'Local', 'value': filename}
 
     # .hhc file
-    hhc = (app.outdir / 'pythondoc.hhc').text()
+    hhc = (app.outdir / 'pythondoc.hhc').read_text()
     tree = HTMLParser(namespaceHTMLElements=False).parse(hhc)
     items = tree.find('.//body/ul')
     assert len(items) == 4
